### DISCUSSION TITLE
open file url

### DISCUSSION THREAD

**Discussion by sekaijin** [elementary/triage#220] [Q&A]
Hi 
file:///.... url open browser not io.elementary.files
howto change it ?

thank

**Comment by jeremypw** [comment]
When typing urls into Terminal and then clicking on them, I find that a url in the form e.g. `file:///home/` opens in Files because it is a folder.  If the url leads to a text file it opens in Code and so on.  Could you give more detail of the circumstances under which the url is opening in the browser and what kind of file it is pointing to?
